slownie.py

```diff
@@ -91,7 +91,9 @@ def slownie999(i):
 def slownie(i, jeden=False, unit=None):
     i = int(i)
     if i < 0:
-        raise ValueError('i < 0')
+        i = i * -1
+        return u' '.join(["minus", slownie(i, jeden, unit)])
```

Comment:
I appreciate the minimalism of the recursive approach, but it leads to suboptimal error messages:
```
>>> slownie.slownie(-10 ** 100)
Traceback (most recent call last):
  File ".../slownie.py", line 115, in slownie
    words = [t, inflect(j, SLOWNIE_1000XX[m])] + words
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".../slownie.py", line 95, in slownie
    return u' '.join(["minus", slownie(i, jeden, unit)])
  File ".../slownie.py", line 117, in slownie
    raise ValueError('i >= 10 ** {n}'.format(n=3 * len(SLOWNIE_1000XX)))
ValueError: i >= 10 ** 66
```